diff --git a/node_modules/@dgreasi/react-native-time-slot-picker/src/TimeSlotPicker.tsx b/node_modules/@dgreasi/react-native-time-slot-picker/src/TimeSlotPicker.tsx
index c14b3c4..8999301 100644
--- a/node_modules/@dgreasi/react-native-time-slot-picker/src/TimeSlotPicker.tsx
+++ b/node_modules/@dgreasi/react-native-time-slot-picker/src/TimeSlotPicker.tsx
@@ -54,14 +54,14 @@ const TimeSlotPicker = ({
 
   // If any changes on date and time selected update data of appointment
   useEffect(() => {
-    if (selectedDate && selectedTime) {
-      setDateOfAppointment({
-        appointmentDate: selectedDate.date,
-        appointmentTime: selectedTime,
-      });
-    } else {
-      setDateOfAppointment(null);
-    }
+    // if (selectedDate && selectedTime) {
+    //   setDateOfAppointment({
+    //     appointmentDate: selectedDate.date,
+    //     appointmentTime: selectedTime,
+    //   });
+    // } else {
+    //   setDateOfAppointment(null);
+    // }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [selectedDate, selectedTime]);
 
@@ -77,7 +77,7 @@ const TimeSlotPicker = ({
       }}
     >
       <View style={{ marginTop }}>
-        <View>
+        {/* <View>
           <ScheduleDatePicker
             selectedDate={selectedDate}
             availableDates={availableDates}
@@ -86,12 +86,17 @@ const TimeSlotPicker = ({
             scheduledAppointment={scheduledAppointment}
             backgroundColor={datePickerBackgroundColor}
           />
-        </View>
+        </View> */}
         {selectedDate && (
           <TimeSlots
             title={timeSlotsTitle}
             selectedTime={selectedTime}
-            setSelectedTime={setSelectedTime}
+            setSelectedTime={(value)=>{
+              setDateOfAppointment({
+                appointmentDate: selectedDate.date,
+                appointmentTime: value,
+              });
+            }}
             slotTimes={selectedDate.slotTimes}
             backgroundColor={timeSlotsBackgroundColor}
           />
diff --git a/node_modules/@dgreasi/react-native-time-slot-picker/src/components/TimeSlots.tsx b/node_modules/@dgreasi/react-native-time-slot-picker/src/components/TimeSlots.tsx
index eaa2b30..e0075a3 100644
--- a/node_modules/@dgreasi/react-native-time-slot-picker/src/components/TimeSlots.tsx
+++ b/node_modules/@dgreasi/react-native-time-slot-picker/src/components/TimeSlots.tsx
@@ -39,7 +39,7 @@ const TimeSlots = ({
   }, [slotTimes, selectedTime]);
 
   return (
-    <View style={[styles.container, { backgroundColor }]}>
+    <View style={[styles.container, { backgroundColor, borderRadius: 8 }]}>
       <Text style={styles.titleContainer}>{title}</Text>
       <View style={styles.timeSlotsContainer}>{getTimeSlots()}</View>
     </View>
